### DISCUSSION TITLE
[Announcement] Homework 0-4 is graded

### DISCUSSION THREAD

**Discussion by KihongHeo** [prosyslab-classroom/cs424-program-reasoning#360] [Announcements]
Hi all,

HW 0-4 is graded. You can see your score (0 to 3) in Gradescope.
The evaluation criteria are the same as in #28.
Most of your essays were really great.

For students who got a non-zero score, please upload your article in [hof.md](https://github.com/prosyslab-classroom/cs492-program-reasoning/blob/main/hof.md) via PR as we did earlier.

Thanks for your hard work.

**Comment by KihongHeo** [comment]
Of course, please remove your student id.

**Comment by park-sunho** [comment]
Hello. Is there any naming format for pdf file? For me, there is already a file named `2022-sunho.pdf` in essay folder, so I need slight variation of filename to upload my new essay there.

**Comment by KihongHeo** [comment]
Hi. Good point. Please name it `2022-synthesis-sunho.pdf`.

**Comment by park-sunho** [comment]
Thank you so much

**Comment by done-4u** [comment]
Hello. Might I modify the previous essay (HW 0-3) while I upload my essay for HW 0-4? I noticed that I did not remove my student ID.

**Comment by KihongHeo** [comment]
@Timber-min sure.
